lib/jbuilder/jbuilder_template.rb

```diff
@@ -161,7 +161,7 @@ def _set_inline_partial(name, object, options)
     elsif _is_collection?(object)
       _scope{ _render_partial_with_options options.merge(collection: object) }
     else
-      locals = ::Hash[options[:as], object]
+      locals = options[:locals].to_h.merge(options[:as].to_sym => object)
```

Comment:
You should rather use `options.fetch(:locals, {}).to_h` ; since this gem still supports Ruby 1.9.3, `nil.to_h` would produce an error. What do you think ?